src/mltk/predictor/gam/GA2MLearner.java

```diff
@@ -68,6 +69,9 @@
 
 		@Argument(name = "-l", description = "learning rate (default: 0.01)")
 		double learningRate = 0.01;
+		
+		@Argument(name = "-p", description = "maximum number of pairs")
```

Comment:
So you would rather modify the interaction file created by FAST to keep only the first `numPairs` lines ? but this mean one would always have to create a new file whenever we want to test another value of `numPairs`